src/storyMap/components/StoryMapsCard.js

```diff
@@ -165,16 +165,25 @@ const StoryMapListItem = props => {
           )}
           <CollaborationIndicator storyMap={storyMap} />
         </Stack>
-        <RouterLink
-          id={`story-map-${storyMap.slug}-link`}
-          to={
-            storyMap.isPublished
-              ? generateStoryMapUrl(storyMap)
-              : generateStoryMapEditUrl(storyMap)
-          }
-        >
-          {storyMap.title}
-        </RouterLink>
+        {/*
+          Unpublished drafts you have not accepted get no link, as it would 404.
+          Published and not accepted get a view link.
+          Published and accepted get an edit link.
+        */}
+        {!storyMap.isPublished && isPending ? (
```

Comment:
Suggestion: I think we can improve this name `isPending` from the name and the component name feels like it should be a pending story map, I think this could be `isMembershipPending`